helm-charts/support/values.yaml

```diff
@@ -24,6 +29,13 @@ prometheus:
   pushgateway:
     enabled: false
   server:
+    ingress:
+      annotations:
+        nginx.ingress.kubernetes.io/auth-type: basic
+        nginx.ingress.kubernetes.io/auth-secret: prometheus-ingress-auth
+        nginx.ingress.kubernetes.io/auth-realm: "Authentication Required"
+        kubernetes.io/ingress.class: nginx
+        cert-manager.io/cluster-issuer: letsencrypt-prod
```

Comment:
This setup enables a nice security feature I think.

By using a netpol for prometheus internally that requires traffic to arrive from the ingress controller pods which in turn enforce passwords, we would have locked down access to prometheus internally and externally.

Reply:
I opened https://github.com/2i2c-org/infrastructure/issues/1101 about this.